docs/source/models/concept_models/vivarium_mncnh_portfolio/antenatal_care_module/module_document.rst

```diff
@@ -42,87 +42,194 @@ Antenatal care attendance module
+  * - B
+    - Attends ANC during first trimester but NOT later pregnancy
+    - ``ANCfirst - min(ANCfirst, ANC4)``
+    - Prevalence of first trimester visit ONLY (and no late pregnancy visit) only occurs if ANCfirst > ANC4
+  * - C
+    - Attends ANC during later pregnancy but NOT first trimester
+    - 
     - 
+  * - D
+    - Does not attend ANC at all during pregnancy
+    - Probability equal to ``ANCfirst – A``  
+    - 
+
+.. note:: 
 
+    As of `pull request #1690 <https://github.com/ihmeuw/vivarium_research/pull/1690>`_ we updated how we assign our ANC propensity 
+    values to include the ANCfirst variable that the HS team processed and shared with us. Please see `these slides <https://uwnetid.sharepoint.com/:p:/r/sites/ihme_simulation_science_team/_layouts/15/Doc.aspx?sourcedoc=%7BADD6223E-9FCA-40BB-BB7F-FE44F377CCDB%7D&file=ANC%20visit%20timing%20data%20strategy%20options.pptx&action=edit&mobileredirect=true>`_ 
+    for more information on this strategy update.
 
-2.3 Module Decision Nodes
------------------------------
+The above propensity values are to be implemented for full term pregnancies only. Partial term pregnancies are assigned 
+propensity values differently because we assume their pregnancies end before they can attend later pregnancy ANC visits. 
+The below table describes what propensity values to use for each exposure option **for partial term pregnancies**:
 
-.. list-table:: Module decision nodes
+.. list-table:: ANC exposure propensity values for partial term pregnancies
   :header-rows: 1
 
-  * - Decision node
+  * - ANC exposure option
     - Description
-    - Information
-    - Note
-  * - 1: Attends ANC?
-    - Compare ANC propensity value input to the ANC1 rate. Ordering of the ANC attendence categories matters: see the "Special ordering of the categories" section on the :ref:`facility choice model document <2024_facility_model_vivarium_mncnh_portfolio>`
-    - ANC1 rates (GBD covariate ID 7): get_covariate_estimates(location_id=location_id, gbd_round_id=7, year_id=2021, decomp_step='iterative', covariate_id=7)
-    - This is location specific, but not age specific. Currently assume that there is no correlation of ANC with other factors. Engineers, you can pull these value straight from GBD, but expected values are as follows - Ethiopia: 75.7%, Nigeria: 74.3%, Pakistan: 90.8%
+    - Propensity value
+    - Notes
+  * - A
+    - Attends ANC during first trimester AND later pregnancy
+    - 0
+    - Assumption
+  * - B
+    - Attends ANC during first trimester but NOT later pregnancy
+    - ``ANCfirst``
+    -
+  * - C
+    - Attends ANC during later pregnancy but NOT first trimester
+    - 0 
+    - Assumption
+  * - D
+    - Does not attend ANC at all during pregnancy
+    - Probability equal to ``1 – ANCfirst``  
+    - 
 
-2.4 Module Action Points
----------------------------
 
-.. list-table:: Module action point
+2.2 Module Inputs
```

Comment:
I think another module input should be the ANC propensity, which will get mapped to the categories A, B, C, D using the probabilities specified here.